products/sle/main.pm

```diff
@@ -980,6 +987,10 @@ elsif (get_var('ISO_IN_EXTERNAL_DRIVE')) {
     load_inst_tests();
     load_reboot_tests();
 }
+elsif (get_var("QAM_SUSECONNECT")) {
```

Comment:
Please document any new variable in the file "variables.md". Why the "QAM" prefix? I think the test module you are trying to add is just as valid for any product validation tests. Also, I recommend to make it an "extra-test" which could be cleaner, e.g. take a look into https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/main_common.pm#L1556 . In this case you would just need to add a function

```perl
sub load_extra_tests_suseconnect {
    loadtest "console/suseconnect";
}
```

as the booting as well as "consoletest_setup" would be covered by the extratest functionality already

Reply:
Changed